pl/mozorg/about/manifesto.lang

```diff
@@ -305,11 +305,11 @@ Zatweetuj
 
 
 ;Learn more about…
-Dowiedz się więcej…
+Więcej informacji…
 
 
 ;Donating to Mozilla
-Donacje na rzecz Mozilli
+Darowizny na rzecz Mozilli
```

Comment:
I don't think this is correct, see also http://bugs.aviary.pl/show_bug.cgi?id=4981